test/test_scaled_matmul_cuda.py

```diff
@@ -1234,6 +1286,199 @@ def test():
         else:
             test()
 
+    @unittest.skipIf(not PLATFORM_SUPPORTS_FP8 or IS_WINDOWS, f8_msg)
+    @unittest.skipIf(not IS_SM90, "cuBLAS blockwise scaling requires sm90+")
+    @unittest.skipIf(
+        _get_torch_cuda_version() < (12, 9),
+        "cuBLAS blockwise scaling added in CUDA 12.9",
+    )
+    @parametrize("output_dtype", [torch.bfloat16, torch.float32])
+    @parametrize("lhs_block,rhs_block", [(1, 1), (128, 1), (1, 128)])
+    @parametrize("M,N,K", [
+        # Nice size
+        (256, 768, 512),
+        # Requires padding for 128x128 scale
+        (384, 128, 1280),
+    ])
+    @parametrize("test_case", [
+        "x_eye_b_eye",
+        "x_ones_y_ones_calc_scales",
+        "x_ones_y_ones_set_scales",
+        "data_random_scales_one",
+    ])
+    def test_scaled_mm_block_wise_numerics(self, output_dtype, lhs_block, rhs_block, M, N, K, test_case):
```

Comment:
just curious, what's the difference between this test and `test_scaled_mm_vs_emulated_block_wise`?

Reply:
The intention is that `test_scaled_mm_vs_emulated_block_wise` is testing a bunch of more "real-world" cases, whereas this is more artificial - cases where we might expect to able to put stricter bounds on errors, and basically convince ourselves that the numerics are now correct.